source/guides/1.0/getting-started.md

```diff
@@ -1078,7 +1078,11 @@ redirect_to routes.books_path
 
 **Congratulations for completing your first Hanami project!**
 
-Let's review what we've done: we've traced requests through Hanami's major frameworks to understand how they relate to each other; we've seen how we can model our domain using entities and repositories; we've seen solutions for building forms, maintaining our database schema, and validating user input.
+Let's review what we've done:
+
+* We've traced requests through Hanami's major frameworks to understand how they relate to each other.
+* We've seen how we can model our domain using entities and repositories.
+* We've seen solutions for building forms, maintaining our database schema and validating user input.
```

Comment:
nice to make it a lis of what has been accomplished, easier to read that way i think,